docs/production/install.md

```diff
@@ -83,6 +97,19 @@ of the failure, you can just rerun the script. For more information, see
   server, i.e., what users will type in their web browser. This becomes
   `EXTERNAL_HOST` in the Zulip [settings][doc-settings].
 
+- `--push-notifications`/`--no-push-notifications`: With this option, the Zulip
+  installer automatically registers your server for the Mobile Push Notification
+  Service and sets up initial default configuration for enabling push notifications.
+  You will prompted to agree to the Terms of Service first. Registration of your server
+  will be run at the end of the installation process.
+
+- `--no-submit-usage-statistics`: If you choose to enable push notifications,
+  by default your server will submit both basic metadata (required for billing
+  and for determining free plan eligibility), as well as aggregate usage
+  statistics. You can disable submitting usage statistics by passing this flag.
+  If push notifications are not enabled, data won't be submitted, so this flag is
+  redundant.
+
```

Comment:
I think these should go below the cert options -- one or the other of the cert options is required, whereas none of these are.